### ISSUE TITLE
Define format of data type 'date' 

### ISSUE THREAD

**Issue by laura-ham** [weaviate/weaviate-semantic-schemas#15]
something like 'yyyy-mm-dd'

**Comment by bobvanluijt** [comment]
Is there not an ISO standard that we should adopt?

**Comment by laura-ham** [comment]
Date, or date and time?

Date:  2017-11-01
Combined date and time in UTC: 2017-11-01T14:28:37+00:00

**Comment by johnciocoiu** [comment]
@laura-ham You are right, datetime format like you said is used: 2017-11-01T14:28:37+00:00
